k6 installation and troubleshooting fixes, add Spanish translation (#646)
grafana/k6-docs: src/data/markdown/translated-guides/en/01 Getting started/02 Installation/01 Troubleshooting.md

> `Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`

To avoid this and be future-proof, users should delete the existing `security@k6.io` repository key on their system, and update their sources list accordingly.

```bash
# delete existing key
sudo apt-key del k6

# import the key the recommended way
sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69

# update the repository
echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
sudo apt-get update
```

## Error importing k6's GPG key

The `gpg` command to import k6's package signing key might fail with:
```bash
gpg: keybox '/usr/share/keyrings/k6-archive-keyring.gpg' created
gpg: failed to create temporary file '/root/.gnupg/.#lk0x000055db689f2310.a86c4b090dc7.7': No such file or directory
gpg: connecting dirmngr at '/root/.gnupg/S.dirmngr' failed: No such file or directory
gpg: keyserver receive failed: No dirmngr
```

> This happens if it's the first time `gpg` is run by that user, so the directory `/root/.gnupg/` doesn't exist yet. You can create it by running `sudo gpg -k` and trying to import the key again.

Reviewer: ```suggestion
This happens if it's the first time that the user runs `gpg` , so the directory `/root/.gnupg/` doesn't exist yet.
To create the directory, run `sudo gpg -k` and try to import the key again.
```
```
